addons/web/static/src/scss/bootstrap_overridden_report.scss

```diff
@@ -1,2 +1,7 @@
 // remove Emoji fonts
 $font-family-sans-serif: o-add-unicode-support-font(("Lucida Grande", Helvetica, Verdana, Arial, sans-serif), 1);
+
+.col-auto{
+    -webkit-box-flex: 1 !important;
+    width: 100% !important;
+}
```

Comment:
- `bootstrap_overridden.scss` file must only contain variable override. They **cannot** contain anything else.
- Can you use the full flex notation so that this is understandable what you are overridding ? `flex: 1 1 auto` is my guess? No need of the `-webkit-box-` prefix. If this is needed then https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/assetsbundle.py#L534 is broken and needs to be restored
- Add a comment in the code to explain?
- I trust you all with the final result.

Reply:
Also, master not before?

Reply:
* moved in a (maybe) more proper place
* it is needed because of no/broken support by wkhtmltopdf
* added some comment as well
* we target master because we don't trust this to be 100% safe for all other reports, in master we would have the time and possibility to ensure that everything is order